src/pages/fieldManagement/settings/defaultFieldComponents/Editors.jsx

```diff
@@ -1,42 +1,62 @@
 import React from 'react';
-import { get } from 'lodash-es';
+import { get, map } from 'lodash-es';
 
 import TreeEditor from './TreeEditor';
 import ConfigureDefaultField from './ConfigureDefaultField';
+import RelationshipEditor from './RelationshipEditor';
 
-export function RelationshipEditor() {
-  return null;
-}
+export function RelationshipEditorWrapper({
+  onClose,
+  onSubmit,
+  formSettings,
+  setFormSettings,
+  children,
+  ...rest
+}) {
+  const relationshipOptions = formSettings
+    ? get(formSettings, 'relationships')
+    : [];
 
-// export function RelationshipEditor({
-//   onClose,
-//   onSubmit,
-//   formSettings,
-//   setFormSettings,
-//   ...rest
-// }) {
-//   const speciesOptions = get(formSettings, 'relationships', []);
+  const transformedRelationshipOptions = map(
+    relationshipOptions,
+    (relationshipArr, key) => {
+      const currentObj = {};
```

Comment:
I doubt that this was necessary in retrospect. But my brain worked much better when the categories/types were tranformed in this way.